hibernate-core/src/main/java/org/hibernate/jpa/boot/internal/EntityManagerFactoryBuilderImpl.java

```diff
@@ -901,6 +901,11 @@ protected void populate(SessionFactoryBuilder sfBuilder, StandardServiceRegistry
 			( ( SessionFactoryBuilderImplementor ) sfBuilder ).disableJtaTransactionAccess();
 		}
 
+		final boolean allowDetachedInstanceRefresh = readBooleanConfigurationValue( org.hibernate.cfg.AvailableSettings.ALLOW_DETACHED_INSTANCE_REFRESH );
+		if(!allowDetachedInstanceRefresh ){
```

Comment:
In the NoORM projects, we changed our ParenPad checkstyle rule to deal with more tokens:
```
<property name="tokens" value="CTOR_CALL, METHOD_CALL, SUPER_CTOR_CALL, LITERAL_FOR, LITERAL_IF,
        LITERAL_WHILE, LITERAL_SWITCH, LITERAL_NEW, "/>
```
(and we're going to add `RESOURCE_SPECIFICATION` soon).

Might be interesting to do the same for ORM.